sqllineage/utils/sqlparse.py

```diff
@@ -128,6 +128,24 @@ def post(tlist, pidx, tidx, nidx):
     )
 
 
+@recurse(Function)
+def group_functions_as(tlist):
```

Comment:
Can you add some comments to help understanding for this function?

I know I should do better for group_window function above, sorry.

Reply:
Added a comment  813598f
Please let me know if it suffices.

Reply:
Yeah, the comment looks good.
I'll spend sometime to further understand this issue during weekend and then get back to you. Forgive me for being maybe over-cautious with monkey patching.

Reply:
Added more strict condition to scope the target [8db1b82](https://github.com/reata/sqllineage/pull/258/commits/8db1b82d23ce151f6d91bb4ec283b8c208eb47b4)